extensions/vscode/webviews/homeView/src/components/tree/TreeItem.vue

```diff
@@ -159,13 +164,24 @@ const toggleExpanded = () => {
         line-height: 22px;
         color: inherit;
         white-space: pre;
+        min-width: 0;
+        overflow: hidden;
+        text-overflow: ellipsis;
       }
 
       .tree-item-description {
+        line-height: 22px;
         font-size: 0.9em;
         margin-left: 0.5em;
-        opacity: 0.7;
+        flex-shrink: 100000;
```

Comment:
This retains the behavior we had before where the description would shrink before the title.